lib/susedistribution.pm

```diff
@@ -257,34 +257,41 @@ sub x11_start_program {
     $args{match_no_wait} //= 0;
     $args{match_timeout} //= 90 if check_var('DESKTOP', 'kde');
 
-    # Start desktop runner and type command there
-    init_desktop_runner($program, $timeout);
-    # With match_typed we check typed text and if doesn't match - retrying
-    # Is required by firefox test on kde, as typing fails on KDE desktop runnner sometimes
-    if ($args{match_typed} && !check_screen($args{match_typed}, 30)) {
-        send_key 'esc';
+    my @runner_tags = qw(desktop-runner-border desktop-runner-suggestions);
+    push @runner_tags 'desktop-runner-plasma-suggestions' if  check_var('DESKTOP', 'KDE');
```

Comment:
there's a comma missing here, also... KDE should be lowercase

Reply:
I am currently fixing a lot of syntax errors, give me a few minutes ^^